lib/pry/repl_file_loader.rb

```diff
@@ -21,12 +21,9 @@ def initialize(file_name)
 
     # Switch to interactive mode, i.e take input from the user
```

Comment:
This comment will be out of date now. Can we comment this implicit behaviour you described in your commit message?